src/main/java/frc/robot/Robot.java

```diff
@@ -23,6 +25,7 @@ public class Robot extends LoggedRobot {
   private Command m_autonomousCommand;
   private RobotContainer m_robotContainer;
 
+  private Timer m_timer;
```

Comment:
Timer needs to be initialized. Other than that, I believe the code is ready to be merged into main.

Reply:
This has been fixed, I'm pulling into main now.